docs/v4-migration-doc.md

```diff
@@ -45,6 +49,31 @@ export HELM_RELEASE_NAME=...
 
 :construction:
 
+### Removing support for Fluent Bit and Fluentd
+
+Te following changes are required in order to switch to OpenTelemetry:
+
+- `fluent-bit.*` has to be removed from `user-values.yaml`
+- `sumologic.logs.collector.allowSideBySide` should be removed from `user-values.yaml`
+- `sumologic.logs.defaultFluentd.*` should be removed from `user-values.yaml`
+- `fluentd.*` should be removed from `user-values.yaml`
+- `sumologic.logs.metadata.provider` should be removed from `user-values.yaml`
+- `sumologic.metrics.metadata.provider` should be removed from `user-values.yaml`
```

Comment:
Is removing these actually `required`? If they're not removed, they'll just be ignored, right?

Reply:
I'm checking only for `provider` values and `fluent-bit.enabled`. I believe this is sufficient